src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/reporting/generator/TransitionalAccommodationReferralReportGenerator.kt

```diff
@@ -29,7 +29,8 @@ class TransitionalAccommodationReferralReportGenerator : ReportGenerator<
         },
         pncNumber = personInfo.tryGetDetails { it.pnc },
         crn = referralData.crn,
-        gender = personInfo.tryGetDetails { it.gender },
+        sex = personInfo.tryGetDetails { it.gender },
+        genderIdentity = personInfo.tryGetDetails { it.profile?.genderIdentity },
```

Comment:
The changes looks ok, but since the story(cas-300) refers 'Referal Summary' screen then i believe the `genderIdentify` needs to be populated as per existing logic below. Could you check Gareth about requirement?

```
genderIdentity = when (personInfoResult.offenderDetailSummary.offenderProfile.genderIdentity) {
        "Prefer to self-describe" -> personInfoResult.offenderDetailSummary.offenderProfile.selfDescribedGender
        else -> personInfoResult.offenderDetailSummary.offenderProfile.genderIdentity
      },

```
      
Refer PeopleController.peopleSearchGet() which is used to display `Referral Summary` page.

Reply:
I spoke with Gareth and we make it consistent with Referral Summary